docs/api-reference/webgl/copy-and-blit.md

```diff
@@ -0,0 +1,163 @@
+# Copy and Blit
+
+`luma.gl` offers a set of methods that copy or blit data from and to Texture and Framebuffer objects. Data can also be copied into Buffer, TypedArray, Images or Urls.
+
+### copyToArray(opts: Object) : TypedArray
+
+Reads data from a `Framebuffer` or `Texture` object into a TypedArray object and returns it. A new TypedArray object is created when not provided. This method requires a sync between CPU and GPU as pixel values are copied from GPU texture memory to CPU Array object memory. This could introduce a delay as it waits for GPU to finish updating the texture. For asynchronous read, check `copyToBuffer` method.
+
+* opts
+
+  Source options
+  * `source` (`Texture` or `Framebuffer`) - This object will be bound and data copied from it.
+  * `x` - (*number*, default: 0) X offset of the area to be copied,
+  * `y` - (*number*, default: 0) Y offset of the area to be copied,
+  * `width` - (*number*, default: source width) The width of the area to be copied,
+  * `height` - (*number*, default: framebuffer height) The height of the area to be copied,
+  * `format` - (*GLenum*, default: GL.RGBA) The format of the data.
+  * `type` - (*GLenum*, default: type of `pixelArray` or `UNSIGNED_BYTE`) The type of the data.
+  * `attachment` - (*GLenum*, default: `COLOR_ATTACHMENT0`) Used to deduced the `type` when not provided.
+
+  Target options
+  * `targetPixelArray` - (*TypedArray*, default: null) Array object, into which data to be copied.
```

Comment:
Sorry for changing my mind but since function is called `copyToArray` maybe just `targetArray`?